src/components/Orders/OrdersStyles.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+
+export const List = styled.ul`
+  width: 100%;
+  li {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  h3 {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 23px;
+  }
+  p {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 19px;
+    color: #747474;
+  }
+`;
+
+export const DetailsContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-right: 20px;
+  .date {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    h3 {
+      font-size: 24px;
+      line-height: 28px;
+      color: #6b068a;
+    }
+    p {
+      font-weight: 400;
+      font-size: 20px;
+      line-height: 23px;
+      color: #252525;
+    }
+  }
+  .title-container {
+    width: 100%;
+  }
+  @media screen and (min-width: 1200px) {
+    padding-right: 70px;
+  }
+  @media screen and (min-width: 768px) {
+    width: 50%;
+  }
+`;
+
+export const Wrapper = styled.div`
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+
+  hr {
+    background: #6b068a;
+    @media screen and (max-width: 768px) {
+      transform: rotate(180deg);
+      width: 100%;
+    }
+  }
+
+  h2 {
+    font-size: 24px;
+    line-height: 28px;
+    align-items: center;
+    color: #6b068a;
+  }
+
+  @media screen and (min-width: 768px) {
+    flex-direction: row;
+  }
+`;
+
+export const WrapperContainer = styled.div`
+  padding: 40px 0;
+`;
+
+export const ListItems = styled.li`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: dotted;
+  border-color: #747474;
+  width: 100%;
+
+  p:first-child {
+    width: 150px;
+  }
+
+  p:last-child {
+    width: 30px;
+  }
+
+  p {
+    font-weight: 500;
+    font-size: 20px;
```

Comment:
Preferably it would be better to create a theme for font sizes

Reply:
I'll do it in the next branch